### PR TITLE
Bugfix: ironfish-cli, unable to run "yarn clean" and "yarn prepack" under Windows

### PR DESCRIPTION
**Pull request by spale777**
Added OS agnostic way of removing build directory

### CODE REVIEW

**Comment by dguenther** [comment]
Could you use the `rimraf` package to do this instead, so we can make the `postpack` script OS-agnostic as well? It should take adding `rimraf` as a devDependency here, then re-running `yarn install` in the root directory to update `yarn.lock`

**Comment by spale777** [comment]
Sure thing, wanted to avoid adding any unnecessary deps but yeah no problem

**Comment by spale777** [comment]
There you go all fixed up

File: ironfish-cli/package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "oclif": "1.16.1",
     "tsc-watch": "4.2.9",
     "typescript": "4.3.4",
-    "yarn": "^1.22.10"
+    "yarn": "^1.22.10",
+    "rimraf": "^3.0.2"
```
Comment (dguenther) [inline ironfish-cli/package.json:23]: Last change, could you alphabetize this import?

**Review by dguenther — CHANGES_REQUESTED** [review]
Nice, thanks for making that change! Just one more nitpick and then this'll be good to go.

File: ironfish-cli/package.json (hunk 1 above)
Comment (spale777) [reply ironfish-cli/package.json]: Yup, done

**Review by dguenther — APPROVED** [review]
Thanks, looks good!

**Comment by spale777** [comment]
@dguenther Why are tests failing for stuff that hasn't been touched ?

**Comment by dguenther** [comment]
The tests pass, it just fails to upload code coverage due to a permissions issue.

**Review by dguenther — APPROVED** [review]

**Comment by dguenther** [comment]
Attempting to fix the permissions issue in #736

**Pull request merged**